charts/kyverno/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+Thank you for installing {{ .Chart.Name }} 😀
+
+Your release is named {{ .Release.Name }}.
+
+Kyverno chart is installed in "default" Pod Security Standards have been automatically deployed and in audit mode.
```

Comment:
"We have installed the "default" profile of Pod Security Standards and set them in audit mode.

Reply:
Done